tests/kernel/threads/usage/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  kernel.threads:
+    arch_exclude: nios2
```

Comment:
why is this not working on nios2?

Reply:
Well.. i copied this from tests/kernel/threads/lifecycle/lifecycle_api/testcase.yaml. So, not sure why this was excluded there.

Reply:
that is irrelevant,  please remove this from here and if there are issues, please document them in a comment.